sdk/core/Azure.Core.Experimental/src/DynamicData.cs

```diff
@@ -1,8 +1,23 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
```

Comment:
I missed this one... is this an attempt to pull DynamicJson up into a higher level abstraction or did this already exist?

Reply:
This has been there the whole time, and it was always the intention to unify DynamicJson under this abstraction.

We've prioritized this as being on the critical path to GA'ing the type and moving it into Azure.Core, since we want to make sure we have this abstraction right before we would GA either type (or the full set of related types).  I'm actually working on a slightly different variant right now, and hope to put up a new draft PR with that in the next couple of days.